rocketmq-docker/4.0.0-incubating/play.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
+
 sudo docker build -t apache/incubator-rocketmq-namesrv:4.0.0-incubating ./namesrv
 
 sudo docker build -t apache/incubator-rocketmq-broker:4.0.0-incubating ./broker
 
 sudo docker run -d -p 9876:9876 --name rmqnamesrv  apache/incubator-rocketmq-namesrv:4.0.0-incubating
 
-sudo docker run -d -p 10911:10911 -p 10909:10909 --name rmqbroker --link rmqnamesrv:rmqnamesrv -e "NAMESRV_ADDR=rmqnamesrv:9876" apache/incubator-rocketmq-broker:4.0.0-incubating
+sudo docker run -d -p 10911:10911 -p 10909:10909 --name rmqbroker --link rmqnamesrv:rmqnamesrv -e "NAMESRV_ADDR=rmqnamesrv:9876" apache/incubator-rocketmq-broker:4.0.0-incubating
```

Comment:
Could you help to update to the latest rocketmq ？

Reply:
see https://github.com/apache/rocketmq-externals/pull/75

Reply:
@vongosling Sure. I would be glad to do that.

Reply:
@vongosling For issue #75, I submitted PR #89.

Reply:
ok, I will close the pr, we could work on another pr together :-)